models/labels/social/persona/ens/labels_three_letter_ens_count.sql

```diff
@@ -0,0 +1,36 @@
+{{config(alias='three_letter_ens_count')}}
```

Comment:
Hi @CreedsCode, please define the model in models/labels/social/labels_social_schema.yml file.
You can follow the pattern in models/labels/social/labels_social_schema.yml

Reply:
Added the corresponding model

Reply:
Yes, you have done it correctly.